src/celeritas/optical/surface/model/GaussianRoughnessModel.cc

```diff
@@ -0,0 +1,73 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/optical/surface/model/GaussianRoughnessModel.cc
+//---------------------------------------------------------------------------//
+#include "GaussianRoughnessModel.hh"
+
+#include "corecel/data/CollectionBuilder.hh"
+#include "celeritas/inp/SurfacePhysics.hh"
+#include "celeritas/optical/CoreParams.hh"
+#include "celeritas/optical/CoreState.hh"
+#include "celeritas/optical/action/ActionLauncher.hh"
+#include "celeritas/optical/surface/TrackSlotExecutor.hh"
+
+#include "GaussianRoughnessExecutor.hh"
+
+namespace celeritas
+{
+namespace optical
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Construct model from surfaces and inputs.
+ */
+GaussianRoughnessModel::GaussianRoughnessModel(
+    SurfaceModelId model,
+    std::vector<PhysSurfaceId> surfaces,
+    std::vector<InputT> const& inputs)
+    : BuiltinRoughnessModel(model, "gaussian", std::move(surfaces))
+{
+    HostVal<GaussianRoughnessData> data;
+    auto build_sigma_alpha = ::celeritas::make_builder(&data.sigma_alpha);
```

Comment:
This should work now that we use C++17:
```suggestion
    auto build_sigma_alpha = CollectionBuilder{&data.sigma_alpha};
```